src/helper_functions.js

```diff
@@ -0,0 +1,47 @@
+_ACTIVITIES = ["Versammlung", "Gastronomie","Lieferdienste", "Rausgehen",
+			"Laeden", "Rausgehen Ausnahme", "Offene Laeden", "Sport draußen",
+			"Freizeiteinrichtungen", "Gottesdienste", "Schulen", "Besondere Gemeinden",
+			"Versammlung Ausnahme", "Sport Ausnahme"];
+_SUBACTIVITIES = [];
+_ACTIVITIES_WITH_MAX_PPL = ["Versammlung"]
+
+_J_UNRESTRICTED = -1;
+_J_PARTIAL_RESTRICTIONS = 1;
+_J_FORBIDDEN = 0;
+
+// We might need something with more states than binary here in case 
+// a) an activity is not covered for this state in our JSON
+// b) if we want to be specific about restricted activities or comments
+async function canGoOut(city, state, activity, number_of_people = -1) {
+
+
+  const legal_table = await _fillJSONContents('../data/json/LegalTabelle.json');
```

Comment:
Please use `/data/json/LegalTabelle.json` (without the ..) so this will work from any page.

Reply:
Done.